src/components/eRACommons.js

```diff
@@ -34,10 +33,9 @@ export const eRACommons = hh(class eRACommons extends React.Component {
   };
 
   authenticateAsNIHFCUser = async (searchArg) => {
-    const currentUserId = Storage.getCurrentUser().dacUserId;
     let isFcUser = await this.verifyUser();
     if (!isFcUser) {
-      await Researcher.getPropertiesByResearcherId(currentUserId).then(
+      await User.getMe().then(
         (response) => isFcUser = this.registerUserToFC(response),
         () => this.setState({ nihError: true })
       );
```

Comment:
The responses are different between these two API calls. You should also be making sure that the `response` sent to `registerUserToFC` doesn't break that method, which I think it will.